csrankings.csv

```diff
@@ -4826,6 +4827,7 @@ George Danezis,University College London,http://www0.cs.ucl.ac.uk/staff/G.Danezi
 George Darmiton da Cunha Cavalcanti,UFPE,www.cin.ufpe.br/~gdcc,0dSAy8IAAAAJ
 George E. Nasr,Lebanese American University,http://www.lau.edu.lb/about/governance-policies/executive-officers/george-nasr.php,S9yBpOMAAAAJ
 George F. Pinder,University of Vermont,http://www.uvm.edu/~cems/?Page=employee/profile.php&SM=employee/_employeemenu.html&EmID=18,NOSCHOLARPAGE
+George H. Chen,Carnegie Mellon University,http://www.andrew.cmu.edu/user/georgech/index.html,O8eqJA4AAAAJ

```

Comment:
Not here. https://www.cs.cmu.edu/directory

Reply:
SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See George's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html)